Classes/common/CDTDocumentRevision.h

```diff
@@ -19,12 +19,14 @@
 #import "CDTChangedObserver.h"
 
 @class TD_RevisionList;
-@class CDTDocumentRevision;
 
 /**
  * Represents a single revision of a document in a datastore.
  */
-@interface CDTDocumentRevision : NSObject <CDTChangedObserver>
+@interface CDTDocumentRevision : NSObject <CDTChangedObserver> {
+   @protected
+    bool _isFullRevision;
+}
```

Comment:
The `@protected` is required so sub-classes can set the property at `-init` time.